lib/msal-browser/src/app/Configuration.ts

```diff
@@ -51,44 +37,28 @@ export type TelemetryOptions = {
  * - tokenRenewalOffsetSeconds    - sets the window of offset needed to renew the token before expiry
  * - navigateFrameWait            - sets the wait time for hidden iFrame navigation
  */
-export type SystemOptions = {
-    loggerOptions?: BrowserLoggerOptions;
+export type BrowserSystemOptions = SystemOptions & {
+    loggerOptions?: LoggerOptions;
```

Comment:
nitpick: Overwrites a generic loggerOptions object? If it is browser specific, the name is very generic. I will check the code as a whole later, this is for traction.

Reply:
This change was reverted in a later PR.